python/plugins/processing/algs/gdal/OgrToPostGis.py

```diff
@@ -210,7 +210,7 @@ def getConnectionString(self, parameters, context):
         if password:
             arguments.append('password=' + password)
         if schema:
-            arguments.append('active_schema=' + schema)
+            arguments.append("active_schema='{}'".format(schema))
```

Comment:
don't we have a quoteIdentifier available somewhere in API?

Reply:
QgsOgrProviderUtils::quotedIdentifier ?